README.md

```diff
@@ -1,66 +1,61 @@
 # WebARonARCore
 
-**An experimental app for Android that lets developers create Augmented Reality (AR) experiences using web technologies.**
+**An experimental browser for Android that lets developers create Augmented Reality (AR) experiences using web technologies. An [iOS version](https://github.com/google-ar/WebARonARKit) is also available.**
+
+<img alt="Spawn-at-Camera example" src="https://github.com/google-ar/three.ar.js/raw/master/examples/screencaps/20170829-arcore-spawnAtCamera-1.gif" style="float: left; object-fit: cover; width: 45%; height: 20em; margin-right: 1em; "><img alt="Spawn-at-Surface example" src="https://github.com/google-ar/three.ar.js/raw/master/examples/screencaps/20170829-arcore-spawnAtSurface-1.gif" style="width: 45%; height: 20em; object-fit: cover;">
 
 **Note:** This is not an official Google product. Nor is it a fully-featured web browser. Nor are the enabling JavaScript APIs standards, or on the standardization path. WebARonARCore is only meant to enable developer experimentation.
 
-An [iOS version](https://github.com/google-ar/WebARonARKit) is also available.
+## Getting started
+ 
+### <a name="InstallingTheARCoreSDK">1. Install the ARCore SDK</a>
 
-## <a name="SupportedDevices">Supported devices</a>
-WebARonARCore is built on top of Android [ARCore](https://developers.google.com/ar), which requires an ARCore-compatible Android device. For best results, we recommend:
+WebARonARCore is built on top of [Android ARCore SDK](https://developers.google.com/ar), which requires a compatible Android device. For best results, we recommend:
```

Comment:
We need to change this line from "best results" to "officially only supported devices" or something.